lib/command/run-workers.js

```diff
@@ -31,22 +35,26 @@ module.exports = async function (workerCount, options) {
 
   const workers = new Workers(numberOfWorkers, config);
   workers.overrideConfig(overrideConfigs);
-  workers.on(event.test.failed, (failedTest) => {
-    output.test.failed(failedTest);
-  });
-
-  workers.on(event.test.passed, (successTest) => {
-    output.test.passed(successTest);
-  });
-
-  workers.on(event.all.result, () => {
-    workers.printResults();
-  });
-
-  try {
-    await workers.bootstrapAll();
-    await workers.run();
-  } finally {
-    await workers.teardownAll();
+  if (Config.plugins && Config.plugins.reRunFailedTest && Config.plugins.reRunFailedTest.enabled === true) {
```

Comment:
Nah, this is unacceptable. It is a plugin so run-workers should not be affected by it.